deployment/ansible/group_vars/all

```diff
@@ -3,3 +3,6 @@ docker_version: "17.*"
 docker_compose_version: "1.15.0"
 
 shellcheck_version: "0.3.*"
+
+aws_profile: "tilejson-io"
+aws_cli_version: "1.11.56"
```

Comment:
I believe the version of terraform we want to specify is `terraform_version: "0.9.11"`

Reply:
Since we are using Docker to run terraform, we no longer need terraform on the VM. The ansible terraform provisioning steps can be removed.